packages/fxa-auth-server/lib/redis.js

```diff
@@ -175,8 +175,8 @@ class FxaRedis {
     return this.redis.get(key);
   }
 
-  set(key, val) {
-    return this.redis.set(key, val);
+  set(key, val, expiryMode, time, setMode) {
+    return this.redis.set(key, val, expiryMode, time, setMode);
```

Comment:
I'm not sure this will work when not all the args are passed. In the node console i get this error with `set('y','z')`:
```
ReplyError: ERR syntax error
    at parseError (/Users/dcoates/code/fxa/packages/fxa-auth-server/node_modules/ioredis/node_modules/redis-parser/lib/parser.js:179:12)
    at parseType (/Users/dcoates/code/fxa/packages/fxa-auth-server/node_modules/ioredis/node_modules/redis-parser/lib/parser.js:302:14) {
  command: { name: 'set', args: [ 'y', 'z', '', '', '' ] }
```

Reply:
Hmm, yeah, might want some default values in there

Reply:
Ah yes, added defaults.

Reply:
```js
this.redis.set(...Array.from(arguments))
```

ought to work

Reply:
or maybe just `...arguments`... i always forget how `arguments` works

Reply:
Well, `...arguments` makes the tests happy.

Reply:
I'm going to use the same format as the zadd below, and capture them as ...args for passthrough.